test/python/circuit/library/test_nlocal.py

```diff
@@ -732,6 +732,18 @@ def test_circular_on_same_block_and_circuit_size(self):
 
         self.assertCircuitEqual(two.assign_parameters(parameters), ref)
 
+    def test_circuit_with_numpy_integers(self):
+        """Test if TwoLocal can be made and drawn from numpy integers"""
+        num_qubits = 6
+        expected_np32 = [(i, i + 1) for i in np.arange(num_qubits - 1, dtype=np.int32)]
+        expected_np64 = [(i, i + 1) for i in np.arange(num_qubits - 1, dtype=np.int64)]
+
+        two_np32 = TwoLocal(num_qubits, "ry", "cx", entanglement=expected_np32)
+        two_np64 = TwoLocal(num_qubits, "ry", "cx", entanglement=expected_np64)
+
+        self.assertTrue(two_np32.draw())
+        self.assertTrue(two_np64.draw())
```

Comment:
Could you exchange this for a more quantitative test? E.g. counting the number of entanglement gates
```python
expected_cx = 3 * num_qubits * (num_qubits - 1) / 2  # full entanglement and 3 reps
self.assertTrue(two_np32.count_ops()['cx'], expected_cx)
```